data/core/tips.json

```diff
@@ -82,17 +82,17 @@
   {
     "type": "snippet",
     "category": "tip",
-    "text": [ "Dangerous place is not always equal to good loot.  Use your head." ]
+    "text": [ "Just because a place is dangerous doesn't mean it's worth looting." ]
   },
   {
     "type": "snippet",
     "category": "tip",
-    "text": [ "Getting in is not a problem.  Think of way out and don't get cornered." ]
+    "text": [ "Always have an escape plan." ]
```

Comment:
Oversimplified IMHO. Perhaps it could be broken into two separate tips?

Reply:
I'll re-word this one again.